ajax/libs/vue-functional-calendar/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "vue-functional-calendar",
+  "title": "Vue.js Functional Calendar",
+  "version": "2.8.69",
+  "description": "Lightweight, high-performance calendar component based on Vue.js",
+  "keywords": [
+     "vue",
+    "calendar",
+    "component",
+    "vue-component",
+    "functional",
+    "date picker",
+    "date range",
+    "range",
+    "multiple calendars",
+    "multiple",
+    "modal calendar",
+    "modal",
+    "inline",
+    "date",
+    "change month",
+    "change year",
+    "picker",
+    "day",
+    "month",
+    "year",
+    "time",
+    "add",
+    "cli",
+    "command",
+    "keys",
+    "keyword",
+    "keywords",
+    "line",
+    "package",
+    "pkg"
+  ],
+  "homepage": "https://github.com/ManukMinasyan/vue-functional-calendar",
+  "repository": {
+     "type": "git",
+    "url": "https://github.com/ManukMinasyan/vue-functional-calendar.git"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Manuk Minasyan",
+    "email": "manuk.minasyan1@gmail.com",
+    "url": "https://github.com/ManukMinasyan"
+  },
+  "copyright": "2020 Manuk Minasyan",
+  "dependencies": {
+    "core-js": "^2.6.5",
+    "vue": "^2.5.21"
+  },
+  "filename": "vue-functional-calendar.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git@github.com:ManukMinasyan/vue-functional-calendar.git",
+    "basePath": "dist/",
+    "files": [
+      "**/*"
+    ]
```

```suggestion
    "fileMap": [
      {
        "basePath": "dist/",
        "files": [
          "**/*"
        ]
      }
    ]
```
Use the correct format for auto-update, please.